third-party/realdds/doc/streaming.md

```diff
@@ -0,0 +1,2 @@
+
+**realsense/\<model\>/\<serial\>/\<stream name\>** of type image - Frames of the selected stream from the camera's server to the user.
```

Comment:
IMU is TBD?

Reply:
Streaming will be filled out with next commit